diff --git a/lib/createSitemapItem.js b/lib/createSitemapItem.js
index af7f01f633c98b4daee3ddb947056679afd4395c..1aecec4a4648a7041400911f1816c387f2d31429 100644
--- a/lib/createSitemapItem.js
+++ b/lib/createSitemapItem.js
@@ -13,6 +13,9 @@ async function getRouteLastUpdatedAt(route) {
     if (route.metadata?.lastUpdatedAt) {
         return route.metadata?.lastUpdatedAt;
     }
+
+    return null;
+
     if (route.metadata?.sourceFilePath) {
         const lastUpdate = await (0, utils_1.getLastUpdate)(route.metadata?.sourceFilePath);
         return lastUpdate?.lastUpdatedAt;
